modules/_labs/button/react/lib/Button.tsx

```diff
@@ -0,0 +1,206 @@
+import * as React from 'react';
+import {colors} from '@workday/canvas-kit-react-core';
+import {focusRing, GrowthBehavior} from '@workday/canvas-kit-react-common';
+import {CanvasSystemIcon} from '@workday/design-assets-types';
+import {ButtonVariant, ButtonSize, ButtonColors, DropdownButtonVariant} from './types';
+import {ButtonContainer, ButtonLabel, ButtonLabelData, ButtonLabelIcon} from './parts';
+
+export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement>, GrowthBehavior {
```

Comment:
Interestingly, `React.HTMLAttributes<HTMLButtonElement>` doesn't seem to contain `disabled`. Should I switch back to `React.ButtonHTMLAttributes<HTMLButtonElement>`?

Reply:
Yes, the latter is just a superset of the former. The generic is just used to type the event handlers